main/ServiceManager/ServiceManager.cpp

```diff
@@ -128,6 +129,10 @@ void ServiceMangerTaskInit()
  */
 void ServiceMangerInit()
 {
+    if (SharedBusInit())
+        ESP_LOGE(TAG, "Failed to Initialize SharedBus.");
```

Comment:
there is no `break` or `return` after the error, so your code will print: 
```
Failed to Initialize SharedBus.
Initialize SharedBus.
``` 
if the initialization fails!

Reply:
fixed.